pkgs/applications/search/recoll/versionawk.patch

```diff
@@ -0,0 +1,12 @@
+diff -ru recoll-1.21.3-orig/filters/rclpdf recoll-1.21.3/filters/rclpdf
+--- recoll-1.21.3-orig/filters/rclpdf	2015-09-28 08:08:15.000000000 +0100
++++ recoll-1.21.3/filters/rclpdf	2015-12-14 21:37:01.984945925 +0000
+@@ -129,7 +129,7 @@
+ {
+     # Test poppler version: at some point before 0.24, poppler began
+     # to properly escape text inside the header (but not the body).
+-    XYZ=`pdftotext -v 2>&1 | awk '/pdftotext/{print $3}'`
++    XYZ=`pdftotext -v 2>&1 | awk '/version/{print $3}'`
+     MAJOR=`echo $XYZ | cut -d. -f 1`
+     MINOR=`echo $XYZ | cut -d. -f 2`
+     escapeheader=1
```

Comment:
joachifm <notifications@github.com> writes:

>> @@ -0,0 +1,12 @@
>> +diff -ru recoll-1.21.3-orig/filters/rclpdf recoll-1.21.3/filters/rclpdf
>> +--- recoll-1.21.3-orig/filters/rclpdf	2015-09-28 08:08:15.000000000 +0100
>> ++++ recoll-1.21.3/filters/rclpdf	2015-12-14 21:37:01.984945925 +0000
>> +@@ -129,7 +129,7 @@
>> + {
>> +     # Test poppler version: at some point before 0.24, poppler began
>> +     # to properly escape text inside the header (but not the body).
>> +-    XYZ=`pdftotext -v 2>&1 | awk '/pdftotext/{print $3}'`
>> ++    XYZ=`pdftotext -v 2>&1 | awk '/version/{print $3}'`
>> +     MAJOR=`echo $XYZ | cut -d. -f 1`
>> +     MINOR=`echo $XYZ | cut -d. -f 2`
>> +     escapeheader=1
>
> On my machine, this change is pointless: it changes the match clause, but you're still matching the same line in the output from `pdftotext -v`. What problem does this solve?

It avoids later corruption of this shell script by the

   "substituteInPlace  $f --replace pdftotext   ${poppler_utils}/bin/pdftotext "

postInstall command.

-- 
Bojan Nikolic          ||          http://www.bnikolic.co.uk